.gitattributes

```diff
@@ -7,3 +7,5 @@
 
 *.png binary
 *.jpg binary
+
+*.ipynb linguist-vendored
```

Comment:
Hm, I think the `linguist-vendored` attribute is correct [1], but you may have to include it on the same line as `*.ipynb text`. Currently, it doesn't seem that the Jupyter Notebooks are being excluded from linguistics, as shown in the attached screenshot of your fork.

[1] https://stackoverflow.com/questions/40659265/using-gitattributes-for-linguist-examples

<img width="1571" alt="Screen Shot 2021-08-30 at 1 20 28 AM" src="https://user-images.githubusercontent.com/10100490/131289009-7d3b5304-ace8-49e0-9842-a59c4bab4992.png">